about/releases/changelogs/0.20.1/index.md

```diff
@@ -0,0 +1,74 @@
+# 0.20.1
+  - Release Date: 2024-07-18
+  - Commit: [b368116](https://github.com/OpenSpace/OpenSpace/commit/c26f9db7d427c9d72f339f745cb2e5fb243ca6aa)
+  - Full changelog: [link](https://github.com/OpenSpace/OpenSpace/releases/tag/releases%2Fv0.20.1)
+
+Download version 0.20.1 for Windows and Mac on the OpenSpace website [installation page](https://openspaceproject.com/version-0201). Below are notes that highlight new content and bug fixes that will be relevant for OpenSpace users.
+
+
+## Features
+- **TileProviderByDate**: A new TileProvider type to select tile providers based on the date. Has been applied to the VIIRS Earth Layers.
+- **Load More Models**: The model loading system now supports models with vertex colors.
+- **Image Sequencing Performance**: Improved framerate during image sequencing.
```

Comment:
@alexanderbock is this correct

Reply:
I'd say "improved performance' instead of "improved framerate".  And maybe "image creation" instead of "image sequencing"?

It's the rate at which we write out frames while outputting frames from a session recording